packages/create-block-tutorial-template/templates/readme.txt.mustache

```diff
@@ -3,8 +3,8 @@
 Contributors:      {{author}}
 {{/author}}
 Tags:              block
-Requires at least: 5.5.1
-Tested up to:      5.6.0
+Requires at least: 5.6.0
```

Comment:
https://developer.wordpress.org/reference/functions/register_block_type_from_metadata/ - actually it was introduced in WP 5.5 so we could leave it as is.